opta/utils/__init__.py

```diff
@@ -4,6 +4,7 @@
 import sys
 from logging import Formatter, Logger, LogRecord
 from logging.handlers import QueueHandler, QueueListener
+from os.path import exists
```

Comment:
because of mocking. If we did not set it like this, the old mocks would have mocked the actual os.path.exist function instead of the version imported in each module